src/pages/AccountSettings/tabs/OktaAccess/hooks/useUpdateOktaConfig.tsx

```diff
@@ -110,26 +104,21 @@ export const useUpdateOktaConfig = ({ provider, owner }: URLParams) => {
     onSuccess: ({ data }) => {
       const parsedData = ResponseSchema.safeParse(data)
       if (!parsedData.success) {
-        return Promise.reject({
+        return rejectNetworkError({
           status: 404,
           data: {},
           dev: 'useUpdateOktaConfig - 404 failed to parse',
-        } satisfies NetworkErrorObject)
+          error: parsedData.error,
+        })
       }
 
       const error = parsedData.data.saveOktaConfig?.error
       if (error) {
-        if (
-          error.__typename === 'ValidationError' ||
-          error.__typename === 'UnauthorizedError' ||
-          error.__typename === 'UnauthenticatedError'
-        ) {
```

Comment:
I removed this error, because after testing if the mutation is successful then error is `null`, so we should show  the toast if `error` is defined.